Test/SqlDom/Only170SyntaxTests.cs

```diff
@@ -10,6 +10,7 @@ public partial class SqlDomTests
         private static readonly ParserTest[] Only170TestInfos =
         {
             new ParserTest170("RegexpTVFTests170.sql", nErrors80: 1, nErrors90: 1, nErrors100: 0, nErrors110: 0, nErrors120: 0, nErrors130: 0, nErrors140: 0, nErrors150: 0, nErrors160: 0),
+            new ParserTest170("JsonIndexTests170.sql", nErrors80: 2, nErrors90: 10, nErrors100: 10, nErrors110: 10, nErrors120: 10, nErrors130: 10, nErrors140: 10, nErrors150: 10, nErrors160: 10),
```

Comment:
fix the number of errors after your last commit. it should be 8 errors now not 10